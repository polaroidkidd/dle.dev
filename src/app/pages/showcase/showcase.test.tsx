import React from "react";
import renderer from "react-test-renderer";
import { ShowCase } from "./index";


test("Render ShowCase Component", () => {
  const tree = renderer.create(
    <ShowCase/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
