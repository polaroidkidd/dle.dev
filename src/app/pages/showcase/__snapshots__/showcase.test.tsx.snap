// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ShowCase Component 1`] = `
<div>
  SHOWCASE CONTAINER
</div>
`;
